server/views/pages/productDependencies.njk

```diff
@@ -5,6 +5,8 @@
 {% from "govuk/components/select/macro.njk" import govukSelect %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
+{% set containerClasses = "wide-screen" %}
```

Comment:
making the diagram appear the full width of the screen as they were a bit squidged